domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/PaymentState.java

```diff
@@ -1,10 +1,13 @@
 package uk.gov.hmcts.cmc.domain.models;
 
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
 import org.hibernate.validator.constraints.NotBlank;
 
 import java.util.Objects;
 import javax.validation.constraints.NotNull;
 
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
 public class PaymentState {
```

Comment:
Shouldn't we delete it? Is this used anywhere?

Reply:
its used to debug comparison failures in unit tests, would prefer to keep